packages/internal-test-helpers/lib/test-cases/application.js

```diff
@@ -45,6 +45,10 @@ export default class ApplicationTestCase extends TestResolverApplicationTestCase
     }
   }
 
+  getController(name) {
```

Comment:
I'm not a great fan of adding these `getTypexxx` APIs. Seems like we will forever be playing whack-a-mole and guessing which ones are in the base class and which are not.

Can you just add it to the `moduleFor` using `getController`? I'd rather see it be a local helper than pushed to the base class.